Developer-Guide.md

```diff
@@ -181,6 +181,7 @@ You MUST choose one of `alpine`, `centos`, `clearlinux`, `euleros`, and `fedora`
 
 > **Note:**
 >
+> - Clearlinux and Euleros distros are not supported on ppc64le arch.
```

Comment:
- s/arch/architecture/.
- Please could you split this (and the note below) into two bullets:

  - The Clearlinux distribution is only supported on the x86_64 architecture.
  - The Euleros distribution is not supported on the ppc64le architecture.

---

But, as an alternative, it might be better to:

1. Add a table like the following to https://github.com/kata-containers/osbuilder/blob/master/README.md

  | | Alpine | CentOS | ClearLinux | EulerOS | Fedora | 
  |--|--|--|--|--|--|
  | **ARM64** | :heavy_check_mark: | :heavy_check_mark: |  | :heavy_check_mark: | :heavy_check_mark: | 
  | **PPC64le** | :heavy_check_mark: | :heavy_check_mark: |  |  | :heavy_check_mark: | 
  | **x86_64** | :heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |

2. Add notes here with a link to the osbuilder table, reminding developers to review it.

The advantage of doing this being that we only define the valid distro/platform combinations in one place, which makes maintenance easier long-term.

wdyt @jcvenegas, @grahamwhaley?

Reply:
One table sounds good. Nice table as well!